### DISCUSSION TITLE
Support the templates/business-processes/Business Process Catalog with Recorded automated tests

### DISCUSSION THREAD

**Discussion by SamkhawajaSKH** [microsoft/dynamics365patternspractices#120] [Ideas]
Hi,
For every L4 process in the templates/business-processes/Business Process Catalog, assign an automated test script that can be editable.
The test script would adhere to the 3 Ws defined for the L4 process (When, Who,& What) 
detailed in https://github.com/microsoft/dynamics365patternspractices/discussions/119
Thanks
Sam

**Comment by rachel-profitt** [comment]
This is something that we are hoping to be able to put into our long-term roadmap. Thanks for your feedback.
